src/Illuminate/Auth/DatabaseUserProvider.php

```diff
@@ -70,10 +70,9 @@ public function retrieveByToken($identifier, $token)
     {
         $user = $this->conn->table($this->table)
```

Comment:
This is surely the same as `$user = $this->conn->table($this->table)->find($identifier);` now?

Reply:
Yes. Definitely